src/plugins/intel_cpu/src/mkldnn_edge.cpp

```diff
@@ -218,8 +218,10 @@ static inline bool isPhycicalMemCompatible(const MemoryDesc& lhsMemDesc, const M
 
 MKLDNNEdge::ReorderStatus MKLDNNEdge::needReorder() {
     bool optimized = false;
-    if (!getInputDesc().isCompatible(getOutputDesc())) {
-        if (isPhycicalMemCompatible(getInputDesc(), getOutputDesc()) && !getParent()->isConstant()) {
+    auto inputPortDesc = getInputPortDesc();
+    auto outPortDesc = getOutputPortDesc();
+    if (!outPortDesc->isCompatible(*inputPortDesc)) {
```

Comment:
minor, can be done later
please put comment why we compare `outPortDesc->isCompatible(*inputPortDesc)` not via versa

Reply:
Done